docs/sources/installation/kubernetes.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Deploy on Kubernetes
+weight: 2
+---
+
+## Deploy Grafana on Kubernetes
+
+This page explains how to install and run Grafana on Kubernetes (K8S). It uses Kubernetes manifests for the setup. If you prefer Helm, refer to the [Grafana Helm community charts](https://github.com/grafana/helm-charts). 
+
+If you are interested in Grafana Enterprise (not Grafana OS), jump to [Deploy Grafana Enterprise on Kubernetes](#deploy-grafana-enterprise-on-kubernetes) section.
+
+#### Create Grafana Kubernetes manifest
+1. Create the file `grafana.yaml`, then paste the following contents in it. 
+
+```yaml
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: grafana
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: local-path
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: grafana
+  name: grafana
+spec:
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      containers:
+        - image: grafana/grafana:latest
+          imagePullPolicy: IfNotPresent
+          name: grafana
+          ports:
+            - containerPort: 3000
+              name: http-grafana
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /robots.txt
+              port: 3000
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 2
+          resources:
+            limits:
+              memory: 4Gi
+            requests:
+              cpu: 100m
+              memory: 2Gi
+          volumeMounts:
+            - mountPath: /var/lib/grafana
+              name: grafana
+      volumes:
+        - name: grafana
+          persistentVolumeClaim:
+            claimName: grafana
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
+spec:
+  ports:
+    - port: 3000
+      protocol: TCP
+      targetPort: http-grafana
+  selector:
+    app: grafana
+  sessionAffinity: None
+  type: LoadBalancer
+```
+
+
+### Send manifest to Kubernetes API server
+
+```bash
+kubectl apply -f grafana.yaml
+```
+
+1. Check that it worked by running the following:
+```bash
+kubectl port-forward service/grafana 3000:3000
+```
+Now if you navigate to `localhost:3000` in your browser, you should see a Grafana login page. 
+
+1. Use `admin` for both the username and password to login.
```

Comment:
is the idea that I had a 1. here and and 1. immediately prior? or should this be a 2.?

Reply:
All numbered list should start with 1. They get rendered correctly. For more information, check the style guide :)